README.md

```diff
@@ -176,36 +176,96 @@ Parameters:
 - type: String
 - example: 659bbda98db7a017ddd9147a
 - description: ADP application id to be updated. It can be obtained from ADP app version page- it is available in url
-4. imagename
+4. appVersion
 - type: String
--  example: ExampleApp
-- description: Image name which should be used while uploading container image to ADP
+- example: 1
+- description: ADP application version which needs to be downloaded.
 5. github_repo
 - type: String
 - example: https://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@github.com/aruba/aruba-iotops-example-ble
+- description: Git repo url detail, where updated json can be pushed
+6. app_bundle_path
+- type: String
+- example: aruba-iotops-example-ble/resource/appbundle/appbundle.json
+- description: App bundle path where downloaded app data can be uploaded
+
+## Jenkins CICD pipeline for App Updation
+Pre requisite for Node on which jenkins pipeline is executed.
+1. Installables required for jenkins pipeline
+  * jq, curl, git, docker, bash
+2. Installables required for example app git repo for docker build
+  * golang, make
+
+Create a new Jenkins pipeline with below mentioned parameters and use the jenkins script available in Jenkinsfile file.
+
+Parameters:
+1. url
+- type: String
+- example: https://pavan257-cl-hybrid-arm-578-0-api.lite.arubadev.cloud.hpe.com
+- description: Central URL required for NBAPI call. It can be obtained from Central NBAPI page
+2. token
+ - type: String
+- example: WjxstwgRaPVOgyhy1JT64o8Mi031WhA7
+- description: Central token required for NBAPI authorization. It can be obtained from Central NBAPI My Apps and token page
+3. appid
+- type: String
+- example: 659bbda98db7a017ddd9147a
+- description: ADP application id to be updated. It can be obtained from ADP app version page- it is available in url
+4. github_repo
+- type: String
+- example: https://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@github.com/aruba/aruba-iotops-example-ble
 - description: Git repo url detail
-6. build_command
+5. app_bundle_path
+- type: String
+- example: aruba-iotops-example-ble/resource/appbundle/appbundle.json
+- description: App bundle path from where app information can be used which can be used while updating ADP app
+6. icon_upload_required
```

Comment:
Can we make it automatic, like uploading icon/Lua/image only when they're changed?  Jenkins `when { changeset }` may work.

Reply:
changeset not working, hence using checkbox approach
and extra parameters are removed now